modules/mpc-party/main.tf

```diff
@@ -247,14 +247,36 @@ resource "kubernetes_service_account" "mpc_party_service_account" {
   depends_on = [kubernetes_namespace.mpc_party_namespace, module.iam_assumable_role_mpc_party]
 }
 
+resource "aws_iam_policy" "connector_aws" {
```

Comment:
We need to make a similar change to `iam_assumable_role_mpc_party`. The policy statement should be something like the below.

Can I push some changes?

```
{
      "Effect": "Allow",
      "Action": [
        "kms:GenerateDataKey",
        "kms:Decrypt"
      ],
      "Resource": "arn:aws:kms:REGION:ACCOUNT-ID:key/KEY-ID"
    }
```